deploy-web/src/styles/index.css

```diff
@@ -4,6 +4,10 @@
   white-space: nowrap;
 }
 
+zag-portal > div {
+  z-index: 1500 !important;
+}
```

Comment:
`zag-portal` is a custom html element?

Reply:
Yeah, cosmoskit puts the modals there with a default `z-index` of `999` which is lower than our header

Reply:
Ok good to know.